commit c66dc8703193a3d23fabedaff24338a3a6434c6b
Author: Trent Mick <trentm@gmail.com>
Date:   2019-04-26T16:20:13-07:00 (5 months ago)
    
    TRITON-1637 DAPI tests fail in hard-filter-volumes-from.test.js

diff --git a/test/algorithms/hard-filter-volumes-from.test.js b/test/algorithms/hard-filter-volumes-from.test.js
index 21e655c..71d1bf9 100644
--- a/test/algorithms/hard-filter-volumes-from.test.js
+++ b/test/algorithms/hard-filter-volumes-from.test.js
@@ -96,7 +96,7 @@ test('filterVolumesFrom() with no getVm set', function (t) {
 
 
 test('filterVolumesFrom() with getVm set', function (t) {
-	common.OPTS.getVm = function (opts, cb) {
+	common.OPTS.getVm = function (opts, _extra, cb) {
 		return (cb(null, VM_LOOKUP[opts.uuid]));
 	};
 
